src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,71 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    static final int NAME_POSITION = 0;
+    static final int NUM_POSITION = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
The `getStatistic` method is handling multiple tasks such as reading, processing, and writing data. Consider breaking it down into smaller methods, each responsible for a single task, to enhance readability and maintainability.